float64 mcu_time
float64 gps_time

float64 rel_pos_n
float64 rel_pos_e
float64 rel_pos_d

float64 rel_pos_length
float64 rel_pos_heading

float64 rel_pos_length_accuracy
float64 rel_pos_heading_accuracy

uint16 status_flags

uint8 gnss_fix_ok
uint8 diff_soln
uint8 rel_pos_valid
uint8 carrier_solution
uint8 is_moving
uint8 ref_pos_miss
uint8 ref_obs_miss
uint8 rel_pos_heading_valid
uint8 rel_pos_normalized